tests/shaded/distributedlog-core-shaded-test/src/test/java/org/apache/bookkeeper/tests/shaded/DistributedLogCoreShadedJarTest.java

```diff
@@ -59,7 +59,7 @@ public void testProtobufIsShaded() throws Exception {
 
     @Test
     public void testProtobufShadedPath() throws Exception {
-        Class.forName("dlshade.com.google.protobuf.Message");
+        Class.forName("org.apache.bookkeeper.shaded.com.google.protobuf.Message");
```

Comment:
Why this? I think there is a duplicate rule for com.google in the build.gradle

Reply:
Good catch- thanks. Its fixed now.